src/app/rosetta/lib/construction.ml

```diff
@@ -908,27 +655,5 @@ let router ~get_graphql_uri_or_error ~logger (route : string list) body =
         |> Errors.Lift.wrap
       in
       Construction_parse_response.to_yojson res
-  | ["hash"] ->
-      let%bind req =
-        Errors.Lift.parse ~context:"Request"
-        @@ Construction_hash_request.of_yojson body
-        |> Errors.Lift.wrap
-      in
-      let%map res =
-        Hash.Real.handle ~env:Hash.Env.real req |> Errors.Lift.wrap
-      in
-      Construction_hash_response.to_yojson res
-  | ["submit"] ->
-      let%bind req =
-        Errors.Lift.parse ~context:"Request"
-        @@ Construction_submit_request.of_yojson body
-        |> Errors.Lift.wrap
-      in
-      let%bind graphql_uri = get_graphql_uri_or_error () in
-      let%map res =
-        Submit.Real.handle ~env:(Submit.Env.real ~graphql_uri) req
-        |> Errors.Lift.wrap
-      in
-      Construction_submit_response.to_yojson res
```

Comment:
why did you remove these endpoints? I think they're still required: https://www.rosetta-api.org/docs/1.4.9/ConstructionApi.html

Reply:
yes, you're right, it's just the response types that are changed

I'll restore those